### ISSUE TITLE
7.2.3-fix Runbook section 10's totals line is stale, and its loss ratio is unmeasurable at orchestrated-session scale

### ISSUE THREAD

**Issue by md-173** [md-173/faf-test-harness#422]
## Description

Two defects in `documentation/operations/harness-runbook.md` section 10, `--udp-drop-percent`, both
found while following it to measure injected loss.

**1. The quoted totals line is missing a field.** The runbook documents it as:

```
game UDP receiver stopped; sender <id> totals: received N, highest sequence H
```

The game actually emits a third field:

```
game UDP receiver stopped; sender 1 totals: received 662, highest sequence 889, discontinuities 172
```

The section also says "Do not use the `gaps` count". `gaps` is the name on two lines, the mid-run
progress line and the per-datagram DEBUG record (`GameUdpReceiver.java:233`), while the totals line
calls the same quantity `discontinuities` (`GameUdpReceiver.java:251`). A reader told to avoid
`gaps` will not obviously connect that to `discontinuities`, which is the one they are most likely
to reach for, since it is sitting in the very line they were sent to read.

**2. The prescribed ratio cannot be measured on an orchestrated run.** The section gives the ratio
`(H - S + 1 - N) / (H - S + 1)` and says that in an orchestrated run the totals line reaches only
the game's own `logs/mockgame.jsonl`. Both true. But a `session` produces roughly 7 seconds of peer
traffic before it has proven its mesh and tears down, which yields a span of about 30 datagrams per
direction. At n = 30 one standard deviation is close to 9 percentage points, so the ratio cannot
distinguish 40% from 25%, and the section's own caveat about uncounted tail drops biases it further
on a short run. The section presents the ratio as the way to read the result without saying it
needs a sample an orchestrated session does not generate.

## Evidence

Verified 2026-09-18, `mock-game-0.3.0-SNAPSHOT-all.jar` at `e9cd9f3`.

Hand-run, two mock-games pointed at each other, sender at 25%, receiver at 0%, held in the lobby
with `--launch-delay-seconds=-1` so traffic runs long enough to measure:

```
S = 0   N = 662   H = 889   span = 890
loss = (890 - 662) / 890 = 25.6%
control direction: received 905, highest sequence 904, discontinuities 0  (zero loss)
```

That is 25.6% against 25% expected, and it confirms the method is sound when the sample is large.

Orchestrated run, `--mock-game-udp-drop-percent=40`, same day, `session: PASS`:

```
B receiving from A:  S = 35  N = 27  H = 66  span = 32  loss = 15.6%
A receiving from B:  S = 34  N = 21  H = 63  span = 30  loss = 30.0%
```

Both below 40, one about 2.8 sigma low, on spans of 32 and 30. The flag demonstrably took effect
(both games logged `dropping 40%`, and `--udp-drop-percent 40` is visible in the launch argv), so
this is a measurement-scale problem, not an injection failure.

For comparison, two earlier hand-runs at 25% that were cut short by the default auto-launch gave
31.9% and 22.3% on about 113 datagrams each. Both within noise, neither convincing alone. That is
the same effect at three times the orchestrated sample size.

## Acceptance criteria

- [ ] The quoted totals line in section 10 includes the `discontinuities` field as emitted.
- [ ] The "do not use" guidance names both spellings and all three sites: `gaps` on the progress
      line and on the per-datagram DEBUG record, `discontinuities` on the totals line, so a reader
      recognises the field whichever line they are looking at.
- [ ] Section 10 states that the ratio needs a large sample, that an orchestrated `session` does not
      produce one, and that magnitude should be confirmed by a hand-run holding the games in the
      lobby with `--launch-delay-seconds=-1`.
- [ ] Section 10 notes what an orchestrated run does establish on its own: the value reaching the
      game's argv and the `dropping N%` line, which is what the pass-through regressions (#322,
      #392) were about.

## Notes

Found during the pre-release functional checks for 0.3.0 (#410, "Before the dispatch"). Everything
in section 10 that concerns the mechanism is correct and was confirmed, including the ordering
rationale, counting from `S`, the per-sender attribution, and that the adapter does not notice the
loss. Only the log-line contract and the sample-size expectation are wrong. Related: #388
(7.2.3-fix) is another runbook correctness fix.
